README.md

```diff
@@ -291,7 +291,7 @@ In the example above, each search alias defined in `myproject.search_aliases` wi
 The purpose is to not accidently overwrite Django's default manager functions with search aliases.
 
 ### SIGNALS
-*Optional:* if it exists, it must be a dictionary (even empty), and will connect to the `post save` and `pre delete` model functions of *all* models using `bungiesearch.managers.BungiesearchManager` as a manager.
+*Optional:* if it exists, it must be a dictionary (even empty), and will connect to the `post save` and `pre delete` model functions of *all* models using `bungiesearch.managers.BungiesearchManager` as a manager. One may also define a signal processor class for more custom functionality by placing the string value of the module path under a key called `CLASS_NAME` in the dictionary value of `SIGNALS` and defining a `setup` method that connects to django signals (signals are connected to each model which uses a BungiesearchManager).
```

Comment:
Could the doc specify that the function needs a `setup` and `teardown` function where the only parameter is `model`.